process_report/process_report.py

```diff
@@ -51,24 +60,24 @@ def load_institute_map() -> dict:
     return institute_map
 
 
-def load_old_pis(old_pi_file):
-    old_pi_dict = dict()
-
+def load_old_pis(old_pi_file) -> pandas.DataFrame:
     try:
-        with open(old_pi_file) as f:
-            for pi_info in f:
-                pi, first_month = pi_info.strip().split(",")
-                old_pi_dict[pi] = first_month
+        old_pi_df = pandas.read_csv(
+            old_pi_file,
+            converters={
```

Comment:
why are you using converters instead of specifying a `dtype` as in the `merge_csv` function?

Reply:
I used converters because specifying `decimal.Decimal` as a dtype would raise the error:
`TypeError: dtype '<class 'decimal.Decimal'>' not understood`
From your comment, I digged at bit more, and it seems `decimal.Decimal` simply isn't a type [actively supported](https://github.com/pandas-dev/pandas/issues/18991#issuecomment-354478790) by the Pandas module, so we must specify it as a converter, not dtype.

In any case, I made sure that the types of the columns were `decimal.Decimal`

Reply:
That's what `pyarrow` provides. A Decimal `dtype`. 

https://github.com/CCI-MOC/process_csv_report/blob/43d43410850a3e6fc752ccec0ad0119a8113dbd5/process_report/process_report.py#L307